src/nccl_ofi_net.c

```diff
@@ -1228,7 +1228,7 @@ int nccl_net_ofi_info_properties(struct fi_info *nic_prov, int dev_id, int num_d
 				 nccl_ofi_properties_t *props)
 {
 	int ret = 0;
-	nccl_ofi_properties_t dev_props = {0};
+	nccl_ofi_properties_t dev_props, err_props = {0};
```

Comment:
ugh; this seems more dangerous.  We should fix the upstream behavior, not do this.

Reply:
Dropped this change from the PR, will figure out a better way to handle it.